backend/server.js

```diff
@@ -229,6 +271,30 @@ sbp('okTurtles.data/set', PUBSUB_INSTANCE, createServer(hapi.listener, {
       } else {
         socket.send(createPushErrorResponse({ message: `No handler for the '${action}' action` }))
       }
+    },
+    [NOTIFICATION_TYPE.SUB] ({ channelID }: SubMessage) {
+      const socket = this
+      const { server } = this
+
+      if (!socket.pushSubscriptionId) return
+      if (!server.pushSubscriptions[socket.pushSubscriptionId]) {
+        delete socket.pushSubscriptionId
+        return
+      }
```

Comment:
Can you add a comment above this to explain why we silently do nothing if this pushSubscriptionId thing isn't defined?

Reply:
Because there's nothing to do? I can add a comment saying that.